tests/unittests/tests-gcoap/Makefile.include

```diff
@@ -0,0 +1,8 @@
+USEPKG += nanocoap
+USEMODULE += gnrc_sock_udp
+
+USEMODULE += gcoap
+USEMODULE += gnrc_udp
+USEMODULE += gnrc_ipv6
```

Comment:
I think there are other tests already that include `gnrc_ipv6` so I'm fine with that (`gcoap` was clear ^^")